CODINGSTYLE.md

```diff
@@ -81,7 +81,7 @@ Prefer american spellings over British spellings, avoid Latin abbreviations.
 ## Code Formatting
 
 - Keep line length, argument count and function size reasonable.
-- Run `make lint-style` command to lint the codebase using a set of style linters.
```

Comment:
Is lint-style not currently in use in the CI? So this is just correction the documentation?

Reply:
`make lint-style` was removed in 2022.  There used to be more lint command in the Makefile `lint`, `lint-style`, `lint-local` with different rulesets.

Github actions doesn't use the Makefile to run the linter.